src/modules/ordersTable/pure/OrdersTableContainer/OrderRow/index.tsx

```diff
@@ -177,8 +178,17 @@ export function OrderRow({
     (executedPriceInverted !== undefined && executedPriceInverted?.equalTo(ZERO_FRACTION)) || withWarning
   const isOrderCreating = CREATING_STATES.includes(order.status)
 
+  const isChildOrder = getIsComposableCowChildOrder(order)
+  const isComposableCowParentOrder = getIsComposableCowParentOrder(order)
+  const isStatusBoxHidden = isComposableCowParentOrder && order.status !== OrderStatus.PRESIGNATURE_PENDING
+
   return (
-    <TableRow data-id={order.id} isOpenOrdersTab={isOpenOrdersTab} isRowSelectable={isRowSelectable}>
+    <TableRow
+      data-id={order.id}
+      isChildOrder={isOpenOrdersTab && isChildOrder}
+      isOpenOrdersTab={isOpenOrdersTab}
```

Comment:
Shouldn't the `isChildOrder` be passed as is and on `TableRow` you check whether it's in the open orders tab?